README.md

```diff
@@ -1,53 +1,70 @@
 # Terraform provider for Scalr
+The Scalr Terraform provider can be used to manage the components within they Scalr IaCP. 
+This will allow you to automate the creation of workspaces, variables, VCS providers and much more.
 
-- Website: https://www.scalr.com/
-
-## Requirements
-
--	[Terraform](https://www.terraform.io/downloads.html) 0.10.x
--	[Go](https://golang.org/doc/install) 1.11 (to build the provider plugin)
-
-## Building The Provider
-
-Clone repository to: `$GOPATH/src/github.com/Scalr/terraform-provider-scalr`
+## Using the provider
+### Requirements
+- [Terraform](https://www.terraform.io/downloads.html) 0.12.x
+Download the latest provider build for your OS and architecture 
+from the [releases page](https://github.com/Scalr/terraform-provider-scalr/releases) 
+that is compatible with your Scalr server version (under the "required" section). 
+Extract the archive to get the provider binary. 
+
+Follow the instructions on the [official documentation page](https://www.scalr.com/docs/en/latest/scalr-terraform-provider/index.html) to learn how to use it.
+## Developing the provider
```

Comment:
I think, developing the provider part may go to the separate doc - where migrations process should be described as well as testing provider locally (incl. `CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build`)